src/modules/fancyzones/lib/FancyZonesDataTypes.cpp

```diff
@@ -112,4 +114,109 @@ namespace FancyZonesDataTypes
             cellRow.resize(m_columns, 0);
         }
     }
+
+    bool DeviceIdData::empty() const
+    {
+        return *this == DeviceIdData{};
+    }
+
+    std::wstring DeviceIdData::Serialize() const
+    {
+        static std::wstring vdId = L"{00000000-0000-0000-0000-000000000000}";
```

Comment:
Should we use `static` in `std::wstring vdId`?
First of all, it's not safe to call this function from multiple threads, second,
if `StringFromCLSID` fails, the function will return the last successful value, not zeroes.
Was this intended?

The easiest fix would be to just remove `static`.

Reply:
Good catch. `static` is a leftover. Removed